webapi/CopilotChat/Extensions/ServiceExtensions.cs

```diff
@@ -89,18 +90,21 @@ public static IServiceCollection AddPersistentOcrSupport(this IServiceCollection
 
         switch (ocrSupportConfig.Type)
         {
+            case OcrSupportOptions.OcrSupportType.AzureFormRecognizer:
+            {
+                services.AddSingleton<IOcrEngine>(sp => new AzureFormRecognizerOcrEngine(ocrSupportConfig.AzureFormRecognizer!.Endpoint!, new AzureKeyCredential(ocrSupportConfig.AzureFormRecognizer!.Key!)));
```

Comment:
The '!' operator implies that neither Endpoint nor Key can be null. But they can!

Reply:
Never mind.  Just saw your option class